log4net.Csv/HeaderOnceAppender.cs

```diff
@@ -0,0 +1,27 @@
+﻿using log4net.Appender;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace log4net.Csv
+{
+    public class HeaderOnceAppender : RollingFileAppender
+    {
+        protected override void WriteHeader()
+        {
+            try
+            {
```

Comment:
Why does this work for guaranteeing the header is only written once?  Can you add a comment or two that explains why this works?  Also, all public members should have xml comment summaries.